subscriber/podaac_data_downloader.py

```diff
@@ -280,6 +280,9 @@ def cmr_downloader(args, token, data_path):
     if args.verbose:
         logging.info(str(results['hits']) + " granules found for " + short_name)  # noqa E501
 
+    if granule is not None and results['hits'] == 0:
+        logging.info("** 0 granules found with -gr (granuleUR) flag; tried using wildcards (*/%)? **")
```

Comment:
a user may takes this one of two ways:
1. are you attempting to use wild cards? 
2. have you tried using wild cards here? it might be helpful?

Recommend changing this to something clearer:

```
0 granules found with -gr (granuleUR) flag; You may want to try using wildcards (*/%) in the granuleUR?
```

Reply:
Gotcha; updated formatting and wording
![Screenshot 2025-03-06 at 10 40 10](https://github.com/user-attachments/assets/b044a6d0-23ed-421a-b407-2c429544787e)